tests/runtime/ray/test_integration.py

```diff
@@ -393,32 +393,6 @@ def test_after_a_failed_task(self, runtime: _dag.RayRuntime):
             inv_output = list(outputs.values())[0]
             assert inv_output == (58,)
 
-        def test_before_any_task_finishes(self, runtime: _dag.RayRuntime, tmp_path):
```

Comment:
This test causes stucks due to the Ray not liking IO operations on Windows, and it tests pretty much redundant to other tests that we have